packages/react-langgraph/src/useLangGraphMessages.ts

```
@@ -72,13 +72,23 @@ export const useLangGraphMessages = <TMessage extends { id?: string }>({
-        } else if (chunk.event === "updates") {
-          setInterrupt(chunk.data.__interrupt__?.[0]);
+        switch (chunk.event) {
+          case LangGraphKnownEventTypes.MessagesPartial:
+          case LangGraphKnownEventTypes.MessagesComplete:
+            setMessages(accumulator.addMessages(chunk.data));
+            break;
+          case LangGraphKnownEventTypes.Updates:
+            setInterrupt(chunk.data.__interrupt__?.[0]);
+            break;
+          case LangGraphKnownEventTypes.Messages:
+            const [messageChunk] = (chunk as LangChainMessageTupleEvent).data;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)